tests/tt_metal/tt_metal/unit_tests_common/compute/matmul/test_matmul_multi_core_multi_dram_in0_mcast_in1_mcast.cpp

```diff
@@ -349,9 +349,18 @@ bool matmul_multi_core_multi_dram_in0_mcast_in1_mcast(tt_metal::Device *device){
     uint32_t M = 16 * num_cores_r;
     uint32_t K = 16 * 12;
     uint32_t N = 16 * num_cores_c;
-    int out_subblock_h = 4;
-    int out_subblock_w = 2;
-    int in0_block_w = 2;
+    int out_subblock_h;
+    int out_subblock_w;
+    int in0_block_w;
+    if (device->arch() == tt::ARCH::BLACKHOLE and not getenv("TT_METAL_DISABLE_BH_ND_WORKAROUND")) {
+        out_subblock_h = 1;
+        out_subblock_w = 1;
+        in0_block_w = 1;
```

Comment:
Do both out_subblock and in0_block_w have to be 1, or is there no hang if only out_subblock or only in0_block_w is 1?

Reply:
yes, I was seeing that both `out_subblock_*` and `in0_block_w` have to be 1 to mitigate the hang

Reply:
Thanks